lib/dialogs/keybindings/index.tsx

```diff
@@ -143,7 +143,7 @@ export class AboutDialog extends Component<DispatchProps> {
                   </li>
                 )}
                 <li>
-                  <Keys keys={[CmdOrCtrl, 'Shift', 'P']}>
+                  <Keys keys={['Ctrl', 'Shift', 'P']}>
                     Toggle Markdown preview
```

Comment:
Is this change meant to address the fact that the Cmd shortcut is bound to something else in the browser? It should still display as Cmd (and this works on Electron).

In fact that reminds me why I'm uncertain about this fix, right now Cmd doesn't work in the browser for some shortcuts, but Ctrl is supported is well (you can use either). So on the Mac you can use Ctrl as a backup, and in the browser you often have to.

Reply:
Ah I see. The only one in the list that I saw that wouldn't work with Cmd in the browser was the Toggle Markdown preview, which is why I updated it to be only `Ctrl`.

I noticed this because `Ctrl + Shift + C` doesn't work for inserting the checklist, only `Cmd + Shift + C` does.

Reply:
@codebykat what do you think the right approach here is?
Should we work to have the insert checklist work with `Ctrl + Shift + C` and only list keyboard shortcuts with the Ctrl key?
Or should we show mostly Cmd key and Ctrl only where Cmd doesn't work?

Reply:
I think the insert checklist should respond to both/either, if possible (I believe that's what the other shortcuts do).

I'm still not sure about how best to document them, to be honest. I wonder if it wouldn't be easier to just document with `CmdOrCtrl` on the Mac and just Ctrl on Windows.

Reply:
The problem being that in the browser, different sets of them work with Cmd (depending on whether the browser overrides them with something else). They all work on Electron.

Reply:
Thanks, @codebykat I've just committed a change that allows inserting the checklist with Ctrl or Cmd + Shift + C on Mac in browser.

For documenting I just want to be sure I understand how you think it should work. 
- On Electron Mac document all keyboard shortcuts using Cmd. 
- On Mac Browser document all keyboard shortcuts using Ctrl.
- On other platforms document all keyboard shortcuts using Ctrl. 

Do I have that right?

There also seem to be some other keyboard shortcuts in the editor which currently only respond to Cmd on Mac browser [like this one](https://github.com/Automattic/simplenote-electron/blob/develop/lib/note-content-editor.tsx#L752). Do we want those to work with Ctrl as well?

Reply:
@SylvesterWilmott Would be curious about your thoughts as well on how we should document keyboard shortcuts across the different platforms.

Reply:
Checked with Sly on this and he agreed that the above way I mentioned above is a good way to go forward. I've updated this now to reflect that.